src/tiled/changetileprobability.cpp

```diff
@@ -29,11 +29,11 @@ namespace Tiled {
 namespace Internal {
 
 ChangeTileProbability::ChangeTileProbability(MapDocument *mapDocument,
-                                             Tile *tile,
-                                             float probability)
+                                             QList<Tile*> tiles,
+                                             QList<float> probabilities)
```

Comment:
Btw, the reason you need a list of probabilities rather than just one is only because it is used to store the previous values. Hence, please just take a single value in the constructor and make the list within this class rather than having the `PropertyBrowser` do that.